README.md

```diff
@@ -128,6 +98,7 @@ Following an example:
 </messageML>
 ```
 
+This is the matching EntityJSON that Zapier sends along with the above MessageML v2:
```

Comment:
It looks like the Zapier application is sending the EntityJSON and that's not true. Integration Bridge builds the EntityJSON parsing the incoming Zapier payload.

Could we change this to something like that: "This is the matching EntityJSON that Zapier parser sends along with the above MessageML v2"?